src/libstd/tests/run-time-detect.rs

```diff
@@ -1,14 +1,6 @@
 //! These tests just check that the macros are available in libstd.
 
-#![cfg_attr(
-    any(
-        all(target_arch = "arm", any(target_os = "linux", target_os = "android")),
-        all(target_arch = "aarch64", any(target_os = "linux", target_os = "android")),
-        all(target_arch = "powerpc", target_os = "linux"),
-        all(target_arch = "powerpc64", target_os = "linux"),
-    ),
-    feature(stdsimd)
-)]
+#![feature(stdsimd)]
```

Comment:
No. But this file tests all features available in both nightly and stable, it makes little sense for it to have such a cfg hell since it requires nightly to run anyways. There is a `ui` test that checks that `is_x86_feature_detected!` works for the stable features on stable Rust here: https://github.com/rust-lang/rust/pull/64534/files#diff-e66d446bef72e3d1dcd7b18783509eeb